fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -155,80 +147,86 @@ def format_df(df: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
-def query_categories(category_keywords: Dict[str, List[str]]) -> pd.DataFrame:
+def create_report1(
```

Comment:
@rohaher Bugfix: The "no messages found" report was also buggy due to the same issue with the "counts report". The logic for it needed to be moved out of `query_keyword()` and into new `create_report1()` function, called near the end of the program execution.